ydb/tests/compatibility/test_export_s3.py

```diff
@@ -9,14 +9,33 @@
 from ydb.tests.oss.ydb_sdk_import import ydb
 from ydb.tests.library.compatibility.fixtures import MixedClusterFixture
 
+from ydb.export import ExportClient
+from ydb.export import ExportToS3Settings
+from ydb.import_client import ImportClient, ImportFromS3Settings
 
-class TestExportS3(MixedClusterFixture):
+
+class TestExportImportS3(MixedClusterFixture):
     @pytest.fixture(autouse=True)
     def setup(self):
         output_path = yatest.common.test_output_path()
         self.output_f = open(os.path.join(output_path, "out.log"), "w")
+        self.prefix = "tables"
+        self.prefix_topics = "topics"
         self.s3_config = self.setup_s3()
-        yield from self.setup_cluster()
+        s3_endpoint, s3_access_key, s3_secret_key, s3_bucket = self.s3_config
+        self.settings = (
+            ExportToS3Settings()
+            .with_endpoint(s3_endpoint)
+            .with_access_key(s3_access_key)
+            .with_secret_key(s3_secret_key)
+            .with_bucket(s3_bucket)
+        )
+
+        yield from self.setup_cluster(
+            extra_feature_flags={
+                "enable_export_auto_dropping": True
```

Comment:
Этого флага нет в 24-4-4, при тестировании совместимости с версией меньшей 25-1 надо бы скипать тест

```
    @pytest.fixture(autouse=True, scope="function")
    def setup(self):
        if min(self.versions) < (25, 1):
            pytest.skip("Only available since 25-1")

        yield from self.setup_cluster(
            extra_feature_flags={
                "enable_follower_stats": True
            }
        )
```